pkg/users/account-create-dialog.js

```diff
@@ -209,9 +211,10 @@ export function account_create_dialog(accounts) {
 
         errors.user_name = validate_username(state.user_name, accounts);
 
-        return password_quality(state.password)
+        return password_quality(state.password, force)
                 .catch(ex => {
-                    errors.password = ex.message || ex.toString();
+                    errors.password = (ex.message || ex.toString()).replace("\n", " ");
+                    errors.password += "\n" + _("Press again to confirm this weak password.");
```

Comment:
Press what again? I think it would be clearer if you say `Press the Apply button again`

Reply:
Or, shorter "Press Apply again"

Reply:
The wording on buttons is `Create` in user create dialog and `Set` in password change dialog.
So should I go with:
- `Press Create again to confirm this weak password.`
- `Press Set again to confirm this weak password.`
?

Reply:
Anaconda uses:
`Press Done again to use the password anyway.`

Reply:
Ok, I went with:
- `Press Create again to use the password anyway.`
- `Press Set again to use the password anyway.`
of course I am happy to change it again.

Reply:
> of course I am happy to change it again.

But only if you ask twice! :wink: